build-apple/Lumia.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,9 @@
 	objects = {
```

Comment:
pls remove files that are in xcode build so that they don't conflict with the configuration on our machines, and only commit files that are in the `source/` folder. i will put them in a .gitignore file soon!